ansible/configs/rhel8lab/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,287 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Mappings:
```

Comment:
Consider using something like this here: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-ec2-template-generate/templates/cloud_template.j2#L5

That way you can take advantage of this healthier list (including cheaper GOLD images) that doesn't have to be maintained in a separate config and get stale.

ami_mappings are in: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-ec2-template-generate/defaults/main.yml